cvat/apps/engine/default_settings.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2024 CVAT.ai Corporation
```

Comment:
Why do we need the file? Why don't define the settings in the global settings file?

Reply:
You can redefine these settings in the global settings file if you need it, this file defines the default values.